packages/contracts/package.json

```diff
@@ -20,27 +20,40 @@
   },
   "license": "MIT",
   "dependencies": {
```

Comment:
For my information, why do we need these dependencies (rather than devDependencies) given that aside from the static JSON config files with contract addresses for Mainnet/Rinkeby, nothing else from the contract package should be needed for a production build?

Reply:
Can probably move them to devDependencies, yea

Reply:
I think this was a bad merge that caused these to move back as I was updating package versions